data/events/scripts/player.lua

```diff
@@ -86,6 +86,25 @@ function Player:onLookInShop(itemType, count)
 end
 
 function Player:onMoveItem(item, count, fromPosition, toPosition, fromCylinder, toCylinder)
+	-- Store Inbox
```

Comment:
I wanna also point out the event Player:onLook, this same discussion could be applied to it but in the end it was moved to Lua so I think this should be in Lua too.